docs/design/flow-monitoring/flow-cache-update.puml

```diff
@@ -0,0 +1,42 @@
+@startuml
+title Flow cache update
+
+participant FlowHS
+control topo.flowhs.flow.monitoring
+participant Server42
+control server42.stats.flowrtt
+box flow-monitoring topology
+participant FlowSpout
+participant FlowLatencySpout
+participant TickBolt
+participant FlowCacheBolt
+participant IslCacheBolt
+participant ActionBolt
+end box
+
+==New path for flow==
+FlowHS -> topo.flowhs.flow.monitoring : new path for flow
+topo.flowhs.flow.monitoring -> FlowSpout : new path for flow
+FlowSpout -> FlowCacheBolt : new path for flow
+FlowCacheBolt -> FlowCacheBolt : update flow cache
+
+==Flow stats==
+Server42 -> server42.stats.flowrtt : FlowRttStatsData
+server42.stats.flowrtt -> FlowLatencySpout : FlowRttStatsData
+FlowLatencySpout -> FlowCacheBolt : FlowRttStatsData
+FlowCacheBolt -> FlowCacheBolt : update flow cache
+
+==Tick==
+TickBolt -> FlowCacheBolt : tick
+activate FlowCacheBolt
+FlowCacheBolt -> ActionBolt : flows with server42 stats
+ActionBolt -> ActionBolt : detect SLA violations
+FlowCacheBolt -> IslCacheBolt : flows without server42 stats
+deactivate FlowCacheBolt
+activate IslCacheBolt
+IslCacheBolt -> IslCacheBolt : calculate actual latency for flows
+IslCacheBolt -> ActionBolt : flows with calculated latency
+deactivate IslCacheBolt
+ActionBolt -> ActionBolt : detect SLA violations
```

Comment:
ok, and what's next?

Reply:
Flow SLA reactions will be discussed and covered by next patches.